packages/create-snap/src/cli.test.ts

```diff
@@ -0,0 +1,78 @@
+import { cli } from './cli';
+import { init } from './cmds';
+
+const getMockArgv = (...args: string[]) => {
+  return ['/mock/path', '/mock/entry/path', ...args];
+};
+
+const HELP_TEXT_REGEX = /^\s*create-metamask-snap \[directory\]/u;
```

Comment:
May need to update this if we decide to update the CLI name.

Reply:
This has been changed to`.+` like we do in the `snaps-cli` ✅